### PR TITLE
Corrections for adapter 3.12 announcement

### PR DESCRIPTION
**Pull request by jnm2**
Followup to https://github.com/nunit/nunit.github.io/pull/59.

### CODE REVIEW

File: _posts/2018-12-19-NUnit3TestAdapter-3.12.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-NUnit3TestAdapter version 3.11.2 is released.
+NUnit3TestAdapter version 3.12.0 is released.
 
-This release focus on two enhancements, generating NUnit3 Test Results xml and embedding the engine to avoid clashes with other packages.
+This release focuses on two enhancements: generating NUnit3 Test Results XML and updating the NUnit engine which avoids clashes with other packages.
```
Comment (jnm2) [inline _posts/2018-12-19-NUnit3TestAdapter-3.12.md]: We didn't embed the engine yet. That would be https://github.com/nunit/nunit3-vs-adapter/issues/583.

File: _posts/2018-12-19-NUnit3TestAdapter-3.12.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-NUnit3TestAdapter version 3.11.2 is released.
+NUnit3TestAdapter version 3.12.0 is released.
 
-This release focus on two enhancements, generating NUnit3 Test Results xml and embedding the engine to avoid clashes with other packages.
+This release focuses on two enhancements: generating NUnit3 Test Results XML and updating the NUnit engine which avoids clashes with user test project dependencies.
```
Comment (jnm2) [inline _posts/2018-12-19-NUnit3TestAdapter-3.12.md]: We didn't embed the engine yet. That would be https://github.com/nunit/nunit3-vs-adapter/issues/583.

**Comment by ChrisMaddock** [comment]
Dropped you a message on Gitter - will leave you to merge this if you want. 😄

**Comment by jnm2** [comment]
Thanks!

**Pull request merged**